src/fonduer/candidates/candidates.py

```diff
@@ -314,16 +322,18 @@ def apply(  # type: ignore
 
                 # Assemble candidate arguments
                 for j, arg_name in enumerate(candidate_class.__argnames__):
-                    candidate_args[arg_name + "_id"] = cand[j][1].id
-
-                # Checking for existence
-                if not clear:
-                    q = select([candidate_class.id])
-                    for key, value in list(candidate_args.items()):
-                        q = q.where(getattr(candidate_class, key) == value)
-                    candidate_id = self.session.execute(q).first()
-                    if candidate_id is not None:
-                        continue
+                    candidate_args[arg_name] = cand[j][1]
+
+                stable_ids = tuple(
```

Comment:
Can we have a global func to construct stable id? It's a little bit messy to have stable id constructor everywhere.

Reply:
I moved these stable_id constructors to `fonduer.utils.utils`.